.github/workflows/code-health.yml

```diff
@@ -16,22 +16,34 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - name: Set up Go
-      uses: actions/setup-go@v5
+    - uses: actions/setup-go@v5
       with:
         go-version-file: 'go.mod'
+    - name: Install tools for mocks and linter
+      run: make tools
+    - name: Mock generation
+      run: mockery
+    - name: Check for uncommited files
+      run: |
+        export FILES=$(git ls-files -o -m --directory --exclude-standard --no-empty-directory)
+        export LINES=$(echo "$FILES" | awk 'NF' | wc -l)
+        if [ $LINES -ne 0 ]; then
+          echo "Detected files that need to be committed:"
+          echo "$FILES" | sed -e "s/^/  /"
+          echo ""
+          echo "Try running: mockery"
```

Comment:
I don't understand this type of message. If there are detected files to be committed, why does it say "try running mockery?"

Reply:
if this step fails is because you forgot to run mockery in local before committing and pushing your changes. what message do you suggest?  (it's similar to the one in CLI)

Reply:
maybe you could add additional context when the step fails like

"It was not possible to autogeneate .... Please, try to run mockery and commit the changes"

Reply:
description changed, let me know if it's better.